UI/window-basic-main.cpp

```diff
@@ -413,12 +413,8 @@ void OBSBasic::UpdateVolumeControlsPeakMeterType()
 
 void OBSBasic::ClearVolumeControls()
 {
-	VolControl *control;
-
-	for (size_t i = 0; i < volumes.size(); i++) {
-		control = volumes[i];
-		delete control;
-	}
+	for (auto& i : volumes)
```

Comment:
Code styling is inconsistent with the rest of the project.  It should be type, space, reference/pointer operator, then variable.  Like this:

```cpp
	for (auto &i : volumes)
```

Also, I would prefer the variable be named something like `vol` rather than `i` if that's okay.  `i` is more of a name you'd use if it were iterating through numbers rather than objects.  `vol` makes it a bit more clear that it's an object.

So I feel:

```cpp
	for (auto &vol : volumes)
		delete vol;
```

would be a bit better.

Additionally, the reference operator isn't even necessary here due to the fact that the vector stores pointers.  You could do:

```cpp
	for (auto vol : volumes)
		delete vol;
```

or, perhaps even:

```cpp
	for (VolControl *vol : volumes)
		delete vol;
```

to make the code a bit more explicit/readable to someone reading the code.